src/styles/blocks/gallery.scss

```
@@ -0,0 +1,118 @@
+.gallery {
+  text-align: center;
+
+  &__title {
+    font-family: "Roboto", sans-serif;
+    font-weight: 700;
+    font-size: 36px;
+    line-height: 36px;
+
+    @media (min-width: 768px) {
+      font-size: 48px;
+      line-height: 48px;
+    }
+  }
+
+  &__block {
+    padding: 16px 0;
+    text-align: left;
+
+    @media (min-width: 768px) {
+      padding: 20px 0;
+    }
+
+    &:first-child {
+      text-align: center;
+    }
+
+    &:last-child {
+      display: flex;
+      justify-content: center;
+    }
+  }
+
+  &__images-wrapper {
+    display: grid;
+    justify-items: center;
+    gap: 20px;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: 130px;
+
+    @media (min-width: 768px) {
+      grid-template-rows: 256px;
+
+      &--1 {
+        grid-template-columns: 2fr 1fr;
+      }
+
+      &--2 {
+        grid-template-columns: repeat(2, 1fr);
+      }
```

```suggestion
      &--2 {
        grid-template-columns: repeat(2, 1fr);
      }
```

Actually, you don't need these styles, because your `.iamges-wrapper` class has already 1fr 1fr